localization/english/decisions/ritual_decisions_l_english.yml

```diff
@@ -6,13 +6,13 @@
 
     ascension_ritual_decision_desc:0 "You have reached the pinnacle of your current Elven level and are ready to ascend!\n\nRetreat to your chambers and begin your meditations. If the Divine Spark is receptive, you will emerge as a higher being."
     ascension_ritual_threshold_1_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! You have achieved the absolute pinnacle form of the Common Elf Tier, and are ready to ascend to become something #italic more #!.\n\nUpon completing this ascension you will enter the ranks of the regal Fae, elves that stand above the common folk, their mandate to rule clear to all."
-    ascension_ritual_threshold_2_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! You have achieved the absolute pinnacle form of the Fae Tier, and are ready to ascend to become something #italic more #!.\n\nUpon completing this ascension you will enter the ranks of the mystical Celestials, angelic elves of great power and true immortality."
-    ascension_ritual_threshold_3_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! You have achieved the absolute pinnacle form of the Celestial Tier, and are ready to ascend to become something #italic more #!.\n\nUpon completing this ascension you will enter the ranks of the divine Ainur, Gods of the Elves and highest servants of the Divine Spark"
+    ascension_ritual_threshold_2_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! You have achieved the absolute pinnacle form of the Fae Tier, and are ready to ascend to become something #italic more #!.\n\nUpon completing this ascension you will enter the ranks of the mystical Celestials, angelic elves of great power and wielding true immortality."
+    ascension_ritual_threshold_3_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! You have achieved the absolute pinnacle form of the Celestial Tier, and are ready to ascend to become something #italic more #!.\n\nUpon completing this ascension you will enter the ranks of the divine Ainur; Gods of the Elves and highest servants of the Divine Spark."
 
     ascension_ritual_decision_confirm:0 "I am ready."
-    ascension_ritual_decision_first_threshold_confirm:0 "I shall be a grand Fae Lord!"
+    ascension_ritual_decision_first_threshold_confirm:0 "I shall become a grand Fae Lord!"
     ascension_ritual_decision_second_threshold_confirm:0 "I shall walk among the Celestials!"
-    ascension_ritual_decision_final_threshold_confirm:0 "I shall become a God who walks among mortals!"
+    ascension_ritual_decision_final_threshold_confirm:0 "I shall become a God who presides over mortals!"
```

Comment:
this one I prefer original.
"I am a god among men" kind of energy, more grandiose